src/lib/stores/namespace.ts

```diff
@@ -1,18 +1,21 @@
 import { browser } from '$app/env';
 import { page } from '$app/stores';
-import { derived } from 'svelte/store';
-import UrlPattern from 'url-pattern';
+import { namespaceUrlPattern } from '$lib/utilities/namespace-url-pattern';
+import { derived, get } from 'svelte/store';
+import { settings } from '$lib/stores/settings';
 
-const pattern = new UrlPattern('/namespaces/:namespace/*');
 const onlyInBrowser = <T>(fn: () => T): T => {
   if (browser) return fn();
 };
 
 export const namespace = derived(page, ($page) => {
-  const match = pattern.match($page.url.pathname);
+  const match = namespaceUrlPattern.match($page.url.pathname);
+
+  const defaultNamespace = get(settings).defaultNamespace;
```

Comment:
Can't we do `$settings.defaultNamespace`?

Reply:
![Screen Shot 2022-03-10 at 2 51 15 PM](https://user-images.githubusercontent.com/1903839/157760671-f95e53d4-6389-449b-8a8a-af7c9b745e2a.png)

Reply:
It gets upset with me when I try. I'd love to use $ is there another way you know of to make that work?

Reply:
I think it's because it's in a ts file and Svelte reactivity isn't enabled in those, unless I'm horribly mistaken (I hope so)